operations/observability/mixins/cross-teams/dashboards.libsonnet

```diff
@@ -14,5 +14,6 @@
     // Example:
     // 'my-new-dashboard.json': (import 'dashboards/components/new-component.json'),
     'gitpod-cluster-autoscaler-k3s.json': (import 'dashboards/gitpod-cluster-autoscaler-k3s.json'),
+    'coredns.json': (import 'dashboards/coredns.json'),
   },
 }
```

Comment:
May I also suggest that we keep this dashboard in the Workspace mixin?

I believe only k3s clusters are using it at the moment